### DISCUSSION TITLE
Source graph - Cody Neovim Setup

### DISCUSSION THREAD

**Discussion by iamshm** [LazyVim/LazyVim#2866] [Q&A]
Hey folks 
I have been trying to setup Cody on Neovim

`:checkheakth sg` gives 
<img width="984" alt="Screenshot 2024-03-28 at 9 27 14 PM" src="https://github.com/LazyVim/LazyVim/assets/43462941/499649f2-fc1a-43df-9406-5e06e1fae55b">

It gives no errors

but still in dont get any suggestion from cody while writing code

Do I need to add `source=cody` to nvim-cmp somewhere? 

can someone please help

**Comment by dusty-phillips** [comment]
This is all I needed:

```lua
return {
  {
    "nvim-cmp",
    dependencies = { "sourcegraph/sg.nvim" },
    opts = function(_, opts)
      table.insert(opts.sources, 1, { name = "cody" })
    end,
  },
  {
    "sourcegraph/sg.nvim",
    dependencies = { "nvim-lua/plenary.nvim", "nvim-telescope/telescope.nvim" },
    opts = {},
  },
}

```
